generate_training_data_scripts/trigger_harvest.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+LANG=en_us_88591
+
+# Get the arguments specified on the command line
+while getopts s:d:e: option
+do
+    case "${option}" in
+       s) START_DATE=${OPTARG};; # Date Format: YYYY-MM-DD (Default: Today's date)
+       d) DATE_DIFF=${OPTARG};; # Specifies how many days to harvest at a time (Default: 2)
+       e) STOP_DATE=${OPTARG};; # Date Format: YYYY-MM-DD (Default: Today's date)
+       \? ) echo "Unknown option: -$OPTARG" >&2; exit 1;;
+       :  ) echo "Missing option argument for -$OPTARG" >&2; exit 1;;
+       *  ) echo "Unimplemented option: -$OPTARG" >&2; exit 1;;
+    esac
+done
+
+# Set default arguments to use if not specified on the command line
+if [ ${OPTIND} == 1 ]
+then
+    STOP_DATE=$(date)
+    STOP_DATE=$(date -d"$STOP_DATE" +%Y-%m-%d)
+    DATE_DIFF=1
+    START_DATE=$(date -d"${STOP_DATE} - $DATE_DIFF day" +%Y-%m-%d)
+fi
+
+# Convert dates to integer like format for easier comparisons
+init_start_date_int=$(date -d"${START_DATE}" +%Y%m%d)
+init_stop_date_int=$(date -d"${STOP_DATE}" +%Y%m%d)
+temp_stop_date=$(date --date="${START_DATE} + ${DATE_DIFF} day" +%Y-%m-%d)
+temp_stop_date=$(date --date="${temp_stop_date} - 1 day" +%Y-%m-%d)
+temp_start_date=$START_DATE
+temp_start_date_int=$(date -d"${temp_start_date}" +%Y%m%d)
+temp_stop_date_int=$(date -d"${temp_stop_date}" +%Y%m%d)
```

Comment:
do you really need all these variables?

Reply:
I think so. For date comparisons, I do need integers. The first variable init_start_date_int is unused so I can remove that.